pkg/artifactory/provider.go

```diff
@@ -85,40 +85,21 @@ func Provider() *schema.Provider {
 				DefaultFunc:  schema.EnvDefaultFunc("ARTIFACTORY_URL", "http://localhost:8082"),
 				ValidateFunc: validation.IsURLWithHTTPorHTTPS,
 			},
-			"username": {
-				Type:          schema.TypeString,
-				Optional:      true,
-				DefaultFunc:   schema.EnvDefaultFunc("ARTIFACTORY_USERNAME", nil),
-				ValidateFunc:  validation.StringIsNotEmpty,
-				ConflictsWith: []string{"api_key"},
-				Deprecated:    "Xray and projects functionality will not work with any auth method other than access tokens (Bearer)",
-			},
-			"password": {
-				Type:          schema.TypeString,
-				Optional:      true,
-				Sensitive:     true,
-				DefaultFunc:   schema.EnvDefaultFunc("ARTIFACTORY_PASSWORD", nil),
-				ConflictsWith: []string{"access_token", "api_key"},
-				ValidateFunc:  validation.StringIsNotEmpty,
-				Deprecated:    "Xray and projects functionality will not work with any auth method other than access tokens (Bearer)",
-				Description:   "Insider note: You may actually use an api_key as the password. This will get your around xray limitations instead of a bearer token",
-			},
 			"api_key": {
 				Type:          schema.TypeString,
 				Optional:      true,
 				Sensitive:     true,
 				DefaultFunc:   schema.EnvDefaultFunc("ARTIFACTORY_API_KEY", nil),
-				ConflictsWith: []string{"username", "access_token", "password"},
+				ConflictsWith: []string{"access_token"},
 				ValidateFunc:  validation.StringIsNotEmpty,
-				Deprecated:    "Xray and projects functionality will not work with any auth method other than access tokens (Bearer)",
+				Description:   "API token. Projects functionality will not work with any auth method other than access tokens",
```

Comment:
So, it was deprecated and now it's not?? verify if a basic user can generate a bearer token. If so, this can go.

Reply:
yes, it's API key. We decided to return it. Basic user can't generate Access key, but can generate API token